tests/arm/Tests.lst

```diff
@@ -35249,7 +35249,7 @@ RelativePath=baseservices\exceptions\regressions\V1\SEH\VJ\ExternalException\Ext
 WorkingDir=baseservices\exceptions\regressions\V1\SEH\VJ\ExternalException
 Expected=0
 MaxAllowedDurationSeconds=600
-Categories=EXPECTED_PASS
+Categories=EXPECTED_PASS;EXPECTED_FAIL;13203
```

Comment:
why `EXPECTED_FAIL`? And how can a test be both `EXPECTED_PASS` and `EXPECTED_FAIL`? And have a bug number?